rfcs/2020-07-21-2744-vicscript.md

```diff
@@ -93,42 +93,28 @@ multiplied = number * 7
 Perform assignments conditionally with an `if` statement:
 
 ```coffee
-sorted_foo = if foo.type() == "array" { foo.sort() }
+sorted_foo = if foo.type() == "array" { foo.sort() } else { foo }
```

Comment:
Does this still represent the conditional assignment that you describe above?

Reply:
Woops, I accidentally changed the purpose of the example. I've switched it back and added more examples.